contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.3.0--2.4.0.sql

```diff
@@ -982,6 +982,23 @@ RETURNS INTEGER AS
 'babelfishpg_tsql', 'object_id'
```

Comment:
do we already have upgrade test for this? Otherwise, we should add as it is minor upgrade use case

Reply:
https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/1097